### DISCUSSION TITLE
how i use in web

### DISCUSSION THREAD

**Discussion by friktiny** [Androz2091/AtlantaBot#486] [Q&A] [answered]
how to use in localhost web like this
![image](https://user-images.githubusercontent.com/76273378/120004086-14fe4400-bff6-11eb-8453-c500d699d0e5.png)

**Comment by Androz2091** [comment, marked as answer]
You can not use Atlanta on replit unless you have a MongoDB server running on another server. In that case, you should listen to the port 3000 (this is configurable in the config file).
